core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/impl/ProjectionRemovealOptimizer.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
```

Comment:
You had this lying around? :)

Reply:
I really need to set up my editor to add this notice for me. Will fix.

Reply:
Fwiw I still get it wrong regularly myself (especially on new unit tests). I wouldn't object to something in the maven formatter  that automatically checks/adds it, but I'm not sure if there is such an option.